perses/annihilation/relative.py

```diff
@@ -135,8 +135,15 @@ def __init__(self,
         self._interpolate_14s = interpolate_old_and_new_14s
 
         #new attributes from the modified geometry engine
-        self.neglected_new_angle_terms = neglected_new_angle_terms
-        self.neglected_old_angle_terms = neglected_old_angle_terms
+        if neglected_old_angle_terms:
```

Comment:
Has this PR accidentally pulled in additional changes?

Reply:
this is an additional change independent of the minimization issue; there was a discrepancy between `self.neglect_`... being an empty list versus a `NoneType`